new-book.md

```diff
@@ -0,0 +1 @@
+* [Coding Projects in Scratch: A Step-by-Step Visual Guide to Coding Your Own Animations, Games](https://www.pdfdrive.com/coding-projects-in-scratch-a-step-by-step-visual-guide-to-coding-your-own-animations-games-simulations-and-more-e185773799.html) (HTML)
```

Comment:
Looks like this is a Paid book. This repository accepts only the resources that are permitted for free use by the license

Reply:
You just have to click on the link and download it , simple

Reply:
In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox.